packages/google_sign_in/google_sign_in_web/example/integration_test/src/person.dart

```diff
@@ -0,0 +1,66 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+const String expectedPersonId = '1234567890';
+const String expectedPersonName = 'Test McTestFace True';
```

Comment:
Actually I ended up renaming our very real person to "Vincent Adultman", because "Testy" sounded like a singular version of something that's defined in the Urban Dictionary (I know "Testy" is a valid english word, but I probably spend too much time in reddit lately :P)